io/pci/pci_hotplug.py

```diff
@@ -119,6 +125,18 @@ def is_removed():
 
         return wait.wait_for(is_removed, timeout=10) or False
 
+    def nvme_splitter_slot(self, pci_addrs):
+        """
+        Returs the physical slot of a nvme splitter drive
+
+        :param pci_addrs: pci_adress of the drive
+        :rtype: string
+        """
+        process.run("drmgr -c pci", ignore_status=True, shell=False)
```

Comment:
do you need this line ? 

also can we keep this in common utils ?

Reply:
This is only specific to splitter.
we might need to write other splitter related utilities based on command activities.
So we will see at that time but for now I feel, to maintain the flow catch the issues we need this fix.
we have observed this issues and every time we are running it manually.

Reply:
got it... so when the issue is fix .. we can remove the line and push it to utils